packages/logging_cloudwatch/amplify_logging_cloudwatch/lib/src/queued_item_store/index_db/indexed_db_adapter.dart

```diff
@@ -138,15 +138,14 @@ class IndexedDbAdapter implements QueuedItemStore {
     await request.future;
 
     for (final elem in request.result) {
-      final value = elem as Map<String, dynamic>;
-      final id = value['id'] as int;
-      final itemValue = value['value'] as String;
-      final timestamp = value['timestamp'] as String;
+      final value = elem as Object;
+      final id = getProperty<int>(value, 'id');
+      final string = getProperty<String>(value, 'value');
```

Comment:
We should avoid naming variables as data types.  String and string can be used interchangeably in some languages making this more confusing.  Why did we rename from itemValue?